lib/features/home_view/home_view.dart

```diff
@@ -32,7 +32,7 @@ class HomeView extends StatelessWidget {
       appBar: LogoAppBar(context),
       body: KeepAliveHomeViewProviders(
         child: ListView.separated(
-          padding: const EdgeInsets.only(bottom: 48),
+          padding: const EdgeInsets.only(bottom: 24),
```

Comment:
Please declare it at `HomeViewConfig `:-)

Reply:
done